plugin-bazel/src/main/kotlin/org/jetbrains/bazel/server/client/BazelClient.kt

```diff
@@ -66,56 +63,56 @@ class BazelClient(
 
     val message = params.message ?: return
 
-    BazelTaskEventsService.getInstance(project).withListener(originId) {
-      onTaskFinish(taskId, maybeParent, message, status, params.data)
-    }
+    // BazelTaskEventsService.getInstance(project).withListener(originId) {
+    //  onTaskFinish(taskId, maybeParent, message, status, params.data)
+    // }
   }
 
   override fun onBuildPublishDiagnostics(params: PublishDiagnosticsParams) {
-    if (!BazelTaskEventsService.getInstance(project).existsListener(params.originId)) {
-      log.debug("Got diagnostics without listener: $params")
```

Comment:
We could probably leave log.debug and remove everything else since it's Intellij specific